src/browse.vue

```diff
@@ -370,13 +377,19 @@ export default {
         },
         searchableFields() {
             return this.tableFields.filter(field => field.searchable).map(field => field.name);
+        },
+        mainDateField() {
```

Comment:
So what happens where there is more than one date field in the filterable fields?

Reply:
It should't be more than one by the moment that's why is called mainDateField; but later, maybe we can allow the User decide which `date field` to use or address the problem in custom filters. 

what do you think?

Reply:
Later is okay.